Connected/Assets/Scripts/Components/LightBulb.cs

```diff
@@ -8,17 +8,14 @@ public class LightBulb : GeneralComponent
     private float requiredPower = 1.0f;
     [SerializeField]
     private float maximumPower = 10f;
+    [SerializeField]
+    private float Ohm = 5f;
```

Comment:
Is there a special reason for breaking naming conventions with the capitalization here?

Reply:
No reason. I just think fields shown in the inspector look better with capitalized first letters. I’ll change it if you prefer non-capitalized first letter. Otherwise I’d personally prefer changing required- and maximumPower to capitalized.

Reply:
Fields shown in the inspector are automatically capitalized and separated into words by Unity when serializing, no need to manually create capitalized names, and thereby break naming convention

Reply:
Thanks for the info.